miden-lib/asm/kernels/transaction/api.masm

```diff
@@ -83,11 +83,11 @@ end
 export.get_account_id
     # get the account id
     exec.account::get_id
-    # => [acct_id, pad(16)]
+    # => [acct_id_hi, acct_id_lo, pad(16)]
 
     # truncate the stack
-    swap drop
-    # => [acct_id, pad(15)]
+    swapw dropw
```

Comment:
This procedure should be only executed during the `syscall` (we `syscall` the `exec_kernel_proc` and then `dynexec` this procedure), and according to ABI the stack should be already padded, so this `dropw` should be safe.

Reply:
Yeah, this is a "read optimization" if we can call it that, since it's a bit easier to read instead of the alternative `movup.2 drop movup.2 drop`. However, as you pointed out, it's also subtle why this is correct and documenting why it is ultimately results in more cognitive load than just having the alternative, so I'll use the alternative.